app/controllers/quotes_controller.rb

```diff
@@ -47,4 +51,12 @@ def destroy
   def quote_params
     params.require(:quote).permit(:body, :quoted_person, :location)
   end
+
+  def vote_types
```

Comment:
Richard wasn't super stoked on it either but it's removed later when I change to using an enum and we utilize I18n functionality.  Obviously can still change it around if desired.